### PR TITLE
sysutil/apuled Monitor IP revision (again)

### PR DESCRIPTION
**Pull request by juliocbc**
@fichtner please check if I did right this time! ;-)

### CODE REVIEW

File: sysutils/apuled/Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-PLUGIN_NAME=		apuled
-PLUGIN_VERSION=		0.1
-PLUGIN_DEVEL=		yes
-PLUGIN_COMMENT=		PC Engine APU LED control
-PLUGIN_MAINTAINER=	julio@cloudfence.com.br
+PLUGIN_NAME=        apuled
```
Comment (fichtner) [inline sysutils/apuled/Makefile]: this still looks off, should be tabs not spaces

File: sysutils/apuled/src/opnsense/scripts/apuled/apuledctl [hunk 2]
```diff
@@ -1,32 +1,34 @@
 #!/bin/sh
+#    Copyright (c) 2019 Cloudfence - Julio Camargo (JCC)
```
Comment (fichtner) [inline sysutils/apuled/src/opnsense/scripts/apuled/apuledctl]: this is also off. second line empty "#", third line down indent is "# " (one space only).

Comment (juliocbc) [reply sysutils/apuled/src/opnsense/scripts/apuled/apuledctl]: Done!

**Comment by fichtner** [comment]
almost but not quite, you can see the differences here: https://github.com/opnsense/plugins/pull/1413/files

**Comment by fichtner** [comment]
Merged manually via 2d0c6ac44327a, thanks!

**Pull request closed without merging**